### DISCUSSION TITLE
Z_0 setting and wind speed oversimulation

### DISCUSSION THREAD

**Discussion by Hon3211** [firemodels/fds#13819] [Q&A]
I'm personally studying FDS, and there's something I don't understand about the Z_0 setting.

![image](https://github.com/user-attachments/assets/a79a3e3e-c451-4bbc-bf79-1549ddf5529c)
![image](https://github.com/user-attachments/assets/a1cfa294-1014-4dfb-b5f9-505b7f5d6f94)

case5 Z_0 value is 0.03

In general, when the Z_0 value increases, the wind speed should decrease, but I wonder if it is not applied in this result.

case4.
![image](https://github.com/user-attachments/assets/5b7a181b-4e80-4db1-87e8-058317ebc1bc)

case5.
![image](https://github.com/user-attachments/assets/40aa9ef2-0172-42d2-9af6-ca39bab1f18d)

Figures are the result of a 2m high wind speed simulation. The domain size is 1200 X 1200 m, and I wonder why the overall wind speed is high.

Even if Z_ref is set to 10m, case 4 shows high wind speed

**Comment by mcgratta** [comment]
I suggest you first create a simple test case with no obstructions, single mesh, etc. Record the velocity profile as a function of height. Then consider the M-O formulae in the User's Guide. That should answer the question.

**Reply by Hon3211** [reply]
Thank you for your reply.
When experimenting with a case without obstructions, wind speed changes according to the M-O formulae.
The problem is that when there are a obstructions, results that do not fit the M-O formula are produced.
I wonder why these results are coming out.

**Reply by mcgratta** [reply]
The M-O profiles represent all those types of near-ground obstructions that one typically cannot resolve in a large-scale simulation of the atmospheric boundary layer. Now you have added explicitly these obstructions, and there is an inherent contradiction. You can use the M-O profiles at the edge of the computational domain to represent the terrain type upwind of the domain, but you cannot expect the profiles in the vicinity of your buildings to match the M-O perfectly.

**Comment by togersen** [comment]
Out of pure curiosity, have you looked into the height of the first cell to achieve a y+ value in the range of 30–100?
